sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/WebPubSubClient.java

```diff
@@ -56,7 +57,7 @@
  * <!-- end com.azure.messaging.webpubsub.client.WebPubSubClient -->
  */
 @ServiceClient(builder = WebPubSubClientBuilder.class)
-public class WebPubSubClient {
+public class WebPubSubClient implements Closeable {
```

Comment:
Make this `final`.

Reply:
Thanks for pointing out. Made all user-facing class `final`.

The `WebPubSubAsyncClient` is invoked by this `WebPubSubClient` (it is Sync over Async under the hood).

`WebPubSubAsyncClient` is currently package private, so user should not see it.